Snabble/UI/DynamicStack/WidgetTextView.swift

```diff
@@ -11,6 +11,20 @@ private protocol WidgetTextStyling {
     var textColor: Color { get }
     var textFont: Font { get }
 }
+private struct NavigationWidget: ViewModifier {
+    let text: String
+    let active: Bool
+    
+    func body(content: Content) -> some View {
+        if active {
+            NavigationLink(text) {
+                content
+            }
+        } else {
+            content
+        }
+    }
+}
```

Comment:
Brauchen wir das nun überhaupt noch, da es nicht zwangsläufig in einem NavigationStack lebt? Die Darstellung müssen wir auf einem anderen Weg umsetzen.

Reply:
af91be43ea04ec977d4df19e4c440af41f7948f0